jOOQ/src/main/java/org/jooq/tools/LoggerListener.java

```diff
@@ -38,6 +38,7 @@
 package org.jooq.tools;
 
 import static java.lang.Boolean.TRUE;
+import static java.util.Objects.requireNonNull;
```

Comment:
Another thing we cannot use in Java 6...

Reply:
Java 8 APIs are allowed in sections that are removed in the Java 6 build. But that's only available in the commercial repositories, not the open source ones. It's a minor thing, which can easily be fixed after merging a PR, don't worry about it too much.